### DISCUSSION TITLE
How to declare the docker host in Kuma ?

### DISCUSSION THREAD

**Discussion by ThierryIT** [wollomatic/socket-proxy#10] [Q&A] [answered]
Hi,
Long time :)
I would like to use Kuma for my running dockers.
For this, I would need to declare the docker host. But how to do this when almost each apps is running is "own" socket proxy ?
Thx

**Comment by wollomatic** [comment, marked as answer]
Hi,

did you see my answer here: https://github.com/wollomatic/socket-proxy/discussions/9#discussioncomment-9062113

I'm not sure if I completly understand what you mean. In most cases a container does not need access to the docker socket. The docker socket ist only needed to exchange information with docker about other containers:
Traefik needs it to be informed about changes in containers regarding the routing.
Watchtower needs access to the socket to be able to update and restart other containers.
Uptime Kuma has a feature to get container information over the docker socket.
But "normal" containers don't need a connection to the docker socket or a socket proxy. They just run.
